plugins/json/src/main/java/org/apache/struts2/json/JSONUtil.java

```diff
@@ -53,7 +68,22 @@
      * @throws JSONException
      */
     public static String serialize(Object object) throws JSONException {
```

Comment:
Not that I can see from just scanning my github fork.  I'll double check when I get home tonight and if not; I'll remove it.